src/config.json

```diff
@@ -15,54 +15,121 @@
+            "description": "The Government of Mexico’s adoption of an approach to climate and disaster risk reduction based in natural solutions is essential to protect its people and infrastructure, and our Coastal Resilience approach and team has a real opportunity to work with the Mexican government and its international policy delegations to influence peer countries in Latin America and emerging economies globally.",
             "mapUrl": "https://coastalresilience.org",
             "image": "assets/images/locations/mexico-centralAm.svg",
             "mapCenter": [17.434510551522894, -96.1962890625],
-            "mapZoom": 3,
+            "mapZoom": 5,
             "subregions": [
                 {
-                    "title": "United States",
-                    "image": "assets/images/locations/US-Mainland.svg",
-                    "mapUrl": "https://coastalresilience.org/"
+                    "title": "Mesoamerican Reef",
+                    "description": "The Mesomerican Reef supports multi-billion dollar tourism and fisheries industries, key income for Mexico, Belize, Guatemala and Honduras. Two million people live along the coast, rapidly developing despite being threatened by tropical storms and sea level rise. Partners in this Coastal Resilience project are working with governments, real state developers, tourism and fisheries sectors to adopt practices that better conserves natural coastlines while reducing people´s vulnerability to climate events.",
+                    "image": "assets/images/locations/mesomerican-reef.svg",
+                    "mapUrl": "http://maps.coastalresilience.org/mar"
                 }
             ]
         },
         {
             "title": "United States",
             "description": "The Coastal Resilience tools provide support for decision-makers working at national and multi-national scales in assessing where to act in risk reduction, adaptation and conservation. They build from critical resources such as the Global Platform on Risk Reduction, World Risk Report, and Conservation Atlas.",
-            "mapUrl": "https://coastalresilience.org",
             "image": "assets/images/locations/US-Mainland.svg",
             "mapCenter": [39.30029918615029, -97.119140625],
             "mapZoom": 3,
             "subregions": [
                 {
-                    "title": "United States",
-                    "image": "assets/images/locations/US-Mainland.svg",
-                    "mapUrl": "https://coastalresilience.org/"
+                    "title": "California",
+                    "description": "Helping California’s coastal communities plan for resilience in the face of sea level rise",
+                    "image": "assets/images/locations/california.svg",
+                    "mapUrl": "https://maps.coastalresilience.org/california"
```

Comment:
These US subregion URLs also don't resolve very quickly if at all.

Reply:
Ah good catch. Shouldn't be `https`.